emulated_hue/api.py

```diff
@@ -298,6 +299,18 @@ async def async_group_action(self, request: web.Request, request_data: dict):
             # forward request to all group lights
             async for entity in self.__async_get_group_lights(group_id):
                 await self.__async_light_action(entity, request_data)
+        if "stream" in group_conf:
+            # Request streaming stop
+            # Duplicate code here. Method instead?
```

Comment:
Let's do that when we refactor the logic into a separate controller instead of in the api code